### ISSUE TITLE
Defect 2b Content Strategy Statement PDF

### ISSUE THREAD

**Issue by srancour** [department-of-veterans-affairs/va.gov-cms#19372]
## Description or Additional Context
The Content Strategy Statement has a downloadable PDF file that consists of a single image that is untagged. This information cannot be reached by the screen reader user.

## Example
The [content strategy statement PDF](https://prod.cms.va.gov/sites/default/files/2021-07/CMS%20knowledge%20base%20content%20strategy%20principles.pdf) that the user can download isn't tagged with anything and is just a flat image. ![Screenshot of the CMS Knowledge Baser content strategy statement title with the image mentioned and the download link to the PDF](https://github.com/user-attachments/assets/44cca643-6b57-412f-a35d-aec8ea6b71d5)

### Code snippet
Link to PDF `<a class="downloadable-file-link downloadable-file-link--document" target="_blank" href="https://prod.cms.va.gov/sites/default/files/2021-07/CMS%20knowledge%20base%20content%20strategy%20principles.pdf" download="https://prod.cms.va.gov/sites/default/files/2021-07/CMS%20knowledge%20base%20content%20strategy%20principles.pdf" aria-label="Download">Download CMS knowledge base content strategy statement  (PDF)</a>`

### Steps to reproduce
Navigate: Home -> Knowledge Base -> How to find Content to Edit -> CMS Knowledge Base content strategy principle
Press download link under image to get PDF.

## Recommended resolution
The PDF being inaccessible is an issue and the PDF should be either remade to not just be a flat image and have proper tagging or it should be removed.

All PDFs in the KB should be checked to make sure that they are accessible.

## Defect Criterion
[1.1.1 Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content) - All [non-text content](https://www.w3.org/TR/WCAG21/#dfn-non-text-content) that is presented to the user has a [text alternative](https://www.w3.org/TR/WCAG21/#dfn-text-alternative) that serves the equivalent purpose, except for the situations listed below.

### Users affected
- Screen reader user

## Teams that this affects:
- [x] CMS Team
- [x] sitewide

**Comment by srancour** [comment]
Talked with Martha and to remediate this we will need to get Adobe Acrobat Pro and CommonLook PDF access, which I requested through yourIT. This may be only possible on a GFE, which I don't have, so we will have to see when the request comes back. If GFE is required, will need to work with @EWashb to get one.

**Comment by EWashb** [comment]
I think we should just remove this PDF and any references to it. That information is outdated.

**Comment by EWashb** [comment]
Thanks Steve. For the additional PDFs you're finding, please create a new ticket for them. This ticket just references the content strategy pdf.

**Comment by srancour** [comment]
New PDF is live, next sprint working on removing the PDF and instead representing in content on the KB page in #19917
